src/rgw/CMakeLists.txt

```diff
@@ -270,33 +270,41 @@ if(WITH_RADOSGW_MOTR)
 endif()
 
 set(rgw_a_srcs
+  librgw.cc
```

Comment:
why do we need librgw.cc in both rgw_a_srcs and librgw_srcs? if it's necessary here in rgw_a, maybe we don't need a separate rgw target, and just make rgw_a the shared lib?

Reply:
it's only supposed to be in rgw_a now, I believe

Reply:
I don't want to change what libs are built, in this PR, if that's ok--I have no specific objection, but I might have if I remembered why we have both

Reply:
okay, if we need it in rgw_a can you please remove librgw.cc from librgw_srcs so we're not compiling it twice?

Reply:
right, agree, that's not intended